internal/wiz/structs.go

```diff
@@ -2295,4 +2345,51 @@ type ClickUpCreateTaskActionTemplateParams struct {
 	ListID string `json:"listId"`
 }
 
-//
+// CreateConnectorInput struct
+type CreateConnectorInput struct {
+	Name        string          `json:"name"`
+	Type        string          `json:"type"` // ID Scalar
+	Enabled     *bool           `json:"enabled,omitempty"`
+	AuthParams  json.RawMessage `json:"authParams"`
+	ExtraConfig json.RawMessage `json:"extraConfig,omitempty"`
+}
+
+// CreateConnectorAuthConfigAWSInput struct
+type CreateConnectorAuthConfigAWSInput struct {
```

Comment:
I don't see this struct in the Wiz API.  If this is a custom struct, it should go into the provider resource file instead.

Reply:
spotted, it was unreferenced and now removed